packages/dnb-design-system-portal/src/docs/uilib/components/global-status/Examples.tsx

```diff
@@ -294,3 +295,85 @@ export const GlobalStatusUpdate = () => (
     }}
   </ComponentBox>
 )
+
+export const GlobalStatusInfoExample1 = () => (
+  <ComponentBox hidePreview hideToolbar>
+    <GlobalStatus id="other-global-status" />
+  </ComponentBox>
+)
+
+export const GlobalStatusInfoExample2 = () => (
+  <ComponentBox hidePreview hideToolbar>
+    <GlobalStatus id="other-global-status" />
+    <Input global_status_id="other-global-status" />
+  </ComponentBox>
+)
+
+export const GlobalStatusInfoExample3 = () => (
+  <ComponentBox hidePreview hideToolbar>
+    <GlobalStatus id="other-global-status" />
+    <FormSet global_status_id="other-global-status">
+      <Input status="Message" />
+    </FormSet>
+  </ComponentBox>
+)
+
+export const GlobalStatusInfoExampleManipulate1 = () => (
+  <ComponentBox hidePreview hideToolbar>
+    {() => {
+      // 1. Update / extend the the status like so:
+
+      const statusOne = GlobalStatus.create({
+        id: 'other-global-status', // or main
+        status_id: 'custom-id-1',
+        text: 'New Text',
+        item: 'Item from status #1',
+      })
+
+      // 2. and removes "custom-id-1" again if needed
+
+      statusOne.update({
+        text: 'Updated Text',
+      })
+
+      // 3. and removes "custom-id-1" again if needed
+
+      statusOne.remove()
+
+      return <></> //ignore this line
```

Comment:
```suggestion
      return <GlobalStatus id="other-global-status" />
```
We could return the component like so. Even in theory, we would have to put `GlobalStatus.create` example coe in a useEffect or outside of the React tree.